crates/fuzzing/src/generators.rs

```diff
@@ -30,16 +32,17 @@ impl fmt::Debug for WasmOptTtf {
     }
 }
 
+#[cfg(feature = "binaryen")]
 impl Arbitrary for WasmOptTtf {
-    fn arbitrary(input: &mut Unstructured) -> arbitrary::Result<Self> {
+    fn arbitrary(input: &mut arbitrary::Unstructured) -> arbitrary::Result<Self> {
         crate::init_fuzzing();
         let seed: Vec<u8> = Arbitrary::arbitrary(input)?;
         let module = binaryen::tools::translate_to_fuzz_mvp(&seed);
```

Comment:
@fitzgen curious on your thoughts on this. An alternative way to handle this would be to do this if `binaryen` is available but otherwise just pull out a `Vec<u8>` and assume it's wasm.

Reply:
(the latter approach reducing the need for `#[cfg]` anywhere basically

Reply:
Two problems with that approach:

1. It may confuse the fuzzer, since the input is interpreted differently in two different cases.
2. We rely on certain properties of the `wasm-opt -ttf` generated Wasm modules, such as that they have a global counter to prevent infinite loops, so it is safe for us to call their exports even though we haven't implemented an interruption API yet.

I think the `#[cfg(...)]`s are the way to go, unfortunately.